test-cases/features/scale-cluster.yaml

```diff
@@ -0,0 +1,20 @@
+test_duration: 1440
+
+cassandra_stress_threads: 500
+cassandra_stress_population_size: 10000000
+space_node_threshold: 100
+
+n_db_nodes: 10
+cluster_target_size: 500
+add_node_cnt: 1
+seeds_num: 1
+#seeds_first: 'true'
```

Comment:
Must have, should be fixed if it has issue.

Reply:
It will be changed after the #1336 will be merged